pkg/virt-config/virt-config.go

```diff
@@ -109,6 +111,13 @@ func IsPPC64(arch string) bool {
 	return false
 }
 
+func IsS390X(arch string) bool {
+	if arch == "s390x" {
+		return true
+	}
+	return false
+}
```

Comment:
nit: simple version `return arch == "s390x"`

Reply:
I applied the nit, but since i originally copied it from the other arches, i changed it there as well to keep it consistent.